lib/components/TextLinkRenderer/TextLinkRenderer.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component, ReactNode, CSSProperties } from 'react';
+import classnames from 'classnames';
+import ThemeConsumer from '../ThemeConsumer/ThemeConsumer';
+import styles from './TextLinkRenderer.css.js';
+
+interface StyleProps {
+  style: CSSProperties;
+  className: string;
+}
+export interface TextLinkRendererProps {
+  children: (styleProps: StyleProps) => ReactNode;
+}
+
+export default class TextLinkRenderer extends Component<TextLinkRendererProps> {
+  static displayName = 'TextLinkRenderer';
+
+  render() {
+    const { children } = this.props;
+
+    return (
+      <ThemeConsumer>
+        {theme =>
+          children({
+            style: {},
```

Comment:
What was the thinking behind this? Is this solely to prevent consumers adding their own `style` overrides?

Reply:
Yep, exactly.